cli/hzn.go

```diff
@@ -3,10 +3,13 @@ package main
 
 import (
 	"flag"
-	"github.com/open-horizon/anax/cli/sdo"
 	"os"
 	"strings"
 
+	"github.com/open-horizon/anax/cli/sdo"
```

Comment:
yeah, these imports are really in an odd order. They should be alphabetic by first letter inside the double quotes.

Reply:
Corrected!